app/scripts/controllers/landing.js

```diff
@@ -44,6 +45,11 @@ angular.module('confRegistrationWebApp')
         };
         $scope.searchComplete = val;
         $scope.searchResults = response.data;
+
+        analytics.digitalData.searchFilter = '';
```

Comment:
@OzzieOrca I changed the Logged In Status data element to custom script and it's now pulling the status. It's no longer relying on SSO Guid to be populated. Will it say "Logged In" no matter which method a user logs in by (FB, relay, etc)?

`if (digitalData.user["0"].profile["0"].profileInfo.loggedInStatus == 'Logged Out') {
      return "Not Logged In";
  } else {
      return "Logged In";
  }`